odsApi_versioned_docs/version-7.2/platform-dev-guide/security/api-profiles.md

```diff
@@ -238,8 +239,43 @@ addresses. If also applied to the `WriteContentType`, the caller will receive a
 error response if they attempt to write anything other than Physical or Shipping
 addresses.
 
+Identifying members of the resource are always included in the `GET` responses
+and must be included in `POST` and `PUT` requests.
+
+```xml
+<!-- Resource-level IncludeOnly -->
+<Profile name="Test-Profile-Resource-IncludeOnly">
+ <Resource name="School">
+  <ReadContentType memberSelection="IncludeOnly">
+   <Property name="NameOfInstitution" />                               <!-- Inherited property -->
+   <Property name="OperationalStatusDescriptor" />                     <!-- Inherited Type property -->
+   <Property name="CharterApprovalSchoolYearTypeReference" />          <!-- Property -->
+   <Property name="SchoolType" />                                      <!-- Type property -->
+   <Property name="AdministrativeFundingControlDescriptor" />          <!-- Descriptor property -->
+   <Collection name="EducationOrganizationAddresses" memberSelection="IncludeAll"/> <!-- Inherited Collection -->
+   <Collection name="SchoolCategories" memberSelection="IncludeAll" /> <!-- Collection -->
+  </ReadContentType>
+  <WriteContentType memberSelection="IncludeOnly">
+   <Property name="ShortNameOfInstitution" />                          <!-- Inherited property -->
+   <Property name="OperationalStatusDescriptor" />                     <!-- Inherited Type property -->
+   <Property name="WebSite" />                                         <!-- Property -->
+   <Property name="CharterStatusType" />                               <!-- Type property -->
+   <Property name="AdministrativeFundingControlDescriptor" />          <!-- Descriptor property -->
+   <Collection name="EducationOrganizationInternationalAddresses" memberSelection="IncludeAll" /> <!-- Inherited Collection -->
+   <Collection name="SchoolGradeLevels" memberSelection="IncludeAll" /> <!-- Collection -->
+  </WriteContentType>
+ </Resource>
+</Profile>
+```
+
+In the example above, the API includes schoolId in `GET` responses, and the API
+client must include it in `POST` and `PUT` requests. Additionally, if required
+fields are excluded, the profile cannot be used to create the resource. In this
```

Comment:
Maybe add emphasis to "create" and augment as follows?

> Additionally, if required
fields are excluded, the profile cannot be used to _create_ the resource _**(though updates would still be possible)**_.